### ISSUE TITLE
Classic Controller Pro analog sticks don't work

### ISSUE THREAD

**Issue by noinkling** [simphax/Touchmote#42]
This has been an issue with a couple of other WiimoteLib-based applications I have tried as well. I did notice you are basing [your build](https://github.com/simphax/WiimoteLib) of WiimoteLib on the 1.7.0.0 version rather than the 1.8.0.0 version that I also see floating around in projects like [WiinUPro](http://www.wiinupro.com/), where the analog sticks do work (too bad the application has other issues). I've tried simply overwriting the dll in Touchmote with the one [found here](https://docs.google.com/viewer?a=v&pid=sites&srcid=ZGVmYXVsdGRvbWFpbnx3aWludXByb3xneDpiZDIyYzE4MDdhNzg1YmI), but then nothing worked, so obviously it's a bit more involved than that. 

There's also this modified version that has support for the Wii U Pro controller, may be of some use: https://github.com/gemisis/WiimoteLib-Pro-Controller-Mod

So basically my question is: what would it take to get the analog sticks on the CC Pro working in Touchmote/Wiimotelib? I'm open to doing a bit of testing if you point me in the right direction.
